src/coordinax/_src/vectors/utils.py

```diff
@@ -30,5 +30,7 @@ def full_shaped(obj: "AbstractVector", /) -> "AbstractVector":
     (2,)
 
     """
-    arrays = jnp.broadcast_arrays(*field_values(obj))
+    from .base import AttrFilter
```

Comment:
We should move `AttrFilter` to avoid having to do this import.

Reply:
I'll just change it to an explicit import from `.base.flags` - forgot to do that earlier